.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+# Description
+
+< Replace with adequate description for this PR as per [Pull Request document](https://www.notion.so/rudderstacks/Pull-Requests-40a4c6bd7a5e4387ba9029bab297c9e3) >
+
+## Linear Ticket
+
+< Replace with Linear Link >
```

Comment:
[optional] I find it very useful because if it's all in a single string I can just double click it and paste over :sweat_smile: 

```suggestion
< Replace_with_Linear_Link >
```